README.adoc

```diff
@@ -0,0 +1,69 @@
+== funops-cli
+
+A library that provides helpers and DRYer import possibilities
+and enhances the following battled tested libraries alread on
+Hackage for our system scripting work in a type safe environment:
+
+* `aeson`
+* `wreq`
+* `doctest`
+* `optparse-applicative`
+* `turtle`
+
+=== Purpose
+
+Provide an API that doesn't require the author of a system CLI script
+in Haskell to know details of `Control.Applicative`, `Control.Monad`,
+`Data.Maybe`, `Data.Aeson`, `Data.Lens`, combinators and where to
+import them from.
+
+=== Assumptions
+
+We work in the Nix environment. So we assume you have Nix package
+manager setup with GHC 7.10.x support.
+
+=== Scope
+
+This library will provide a simpler API for the purposes of writing
+system scripts for the common use cases of:
```

Comment:
I disagree with this comment completely. 

1. The original purpose in the previous version of this PR was utter shit IMO.
2. "Simpler" is qualified by the areas of focus in the bullet points and it is measurable (e.g. how many different things the user of the API has to juggle in their heads to use it, or offering better consistency usually at the expense of API flexibility like I plan in the `optparse-applicative` simplification).